app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -49,15 +52,25 @@ public function __construct()
     protected function validator(array $data)
```

Comment:
Can this be removed? I think `RegistersUser@register` is using this validator though. Please test if okay. 
If not okay, we can override `RegistersUser@register` directly to `RegisterController@register` itself, just copy the logic.